src/test/java/pages/LogInPage.java

```diff
@@ -0,0 +1,48 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LogInPage extends BasePage{
+    public LogInPage(WebDriver givenDriver) {
+
+        super(givenDriver);
+    }
+    //web elements
+    //find elements by using @FindBy methos brovided by Page factori
+    @FindBy(css = "[type = 'submit']")
+    static WebElement submitButton;
+    @FindBy(css= "[type='email']")
+    static WebElement emailField;
+
+    @FindBy(css = "[type = 'password']")
+    static WebElement passwordField;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```